src/Repository/SignalementRepository.php

```diff
@@ -607,29 +611,29 @@ public function findSignalementAffectationIterable(User $user, array $options):
             GROUP_CONCAT(DISTINCT desordreCategories.label SEPARATOR :group_concat_separator_1) as listDesordreCategories,
             GROUP_CONCAT(DISTINCT desordreCriteres.labelCritere SEPARATOR :group_concat_separator_1) as listDesordreCriteres,
             GROUP_CONCAT(DISTINCT tags.label SEPARATOR :group_concat_separator_1) as etiquettes,
-            GROUP_CONCAT(IFNULL(i.occupantPresent, \'-\') ORDER BY i.scheduledAt ASC SEPARATOR :concat_separator) as interventionOccupantPresent,
-            GROUP_CONCAT(IFNULL(i.concludeProcedure, \'-\') ORDER BY i.scheduledAt ASC SEPARATOR :concat_separator) as interventionConcludeProcedure,
-            \'-désactivation temporaire-\' as interventionDetails,
-            GROUP_CONCAT(
-                DISTINCT
-                CONCAT(
-                    i.status,
-                    :concat_separator,
-                    i.scheduledAt
-                )
-                ORDER BY i.scheduledAt ASC
-                SEPARATOR :concat_separator
-            ) as interventionsData
+            MAX(vli.occupantPresent) AS interventionOccupantPresent,
+            MAX(vli.concludeProcedure) AS interventionConcludeProcedure,
+            MAX(vli.details) AS interventionDetails,
+            MAX(vli.status) AS interventionStatus,
+            MAX(vli.scheduledAt) AS interventionScheduledAt,
+            MAX(vli.nbVisites) AS interventionNbVisites
             '
         )->leftJoin('s.situations', 'situations')
             ->leftJoin('s.criteres', 'criteres')
             ->leftJoin('s.desordreCategories', 'desordreCategories')
             ->leftJoin('s.desordreCriteres', 'desordreCriteres')
             ->leftJoin('s.tags', 'tags')
+            ->leftJoin(ViewLatestIntervention::class, 'vli', 'WITH', 'vli.signalementId = s.id')
             ->setParameter('concat_separator', SignalementAffectationListView::SEPARATOR_CONCAT)
             ->setParameter('group_concat_separator_1', SignalementExport::SEPARATOR_GROUP_CONCAT);
 
-        return $qb->getQuery()->toIterable();
+        $iterableResult = $qb->getQuery()->toIterable();
+        foreach ($iterableResult as $key => $row) {
+            yield $row;
+            if (($key % 100) === 0) {
+                gc_collect_cycles();
+            }
```

Comment:
Forcer php à aller checker le garbage collector pour libérer de la mémoire. 
> If you call the function gc_collect_cycles(), then collection of cyclic references is triggered explicitly even if you don’t have 10000 of them in memory yet.
https://tideways.com/profiler/blog/how-to-optimize-the-php-garbage-collector-usage-to-improve-memory-and-performance

Tentative d'usage des méthodes de l'entity manager mais sans succès : 
* ` $entityManager->clear()` me provoque des effets de bord 
* `$entityManager->detach()` travaille qu'avec des objets, pas possible de lui passer un tableau qui le cas de `findSignalementAffectationIterable`

Reply:
Ok mais faudrait que je refasse un test sur le 13, c'est celui qui me posait problème